sw/host/opentitanlib/src/bootstrap/mod.rs

```diff
@@ -77,36 +123,36 @@ impl Bootstrap {
                 unimplemented!();
             }
         };
-        Ok(Bootstrap {
-            protocol,
+        let myself = Bootstrap {
```

Comment:
might be wise to give this a more descriptive name (like `bootstrap`) even if we are local to the `Bootstrap` type.

Reply:
I have factored this method, such that I can use the "self" reserved word.